### PR TITLE
75268 stub new travel claim app

### PR DESCRIPTION
**Pull request by wullaski**
## Summary

- Creates a new application for travel only flow

## Related issue(s)

- [department-of-veterans-affairs/va.gov-team#75268](https://github.com/department-of-veterans-affairs/va.gov-team/issues/75268)

## Testing done

- Visual
- Unit

## What areas of the site does it impact?

/my-health/travel-claim

## Acceptance criteria

- [x] [FE] create app in the check-in directory src/applications/check-in/travel-claim
- [x] [FE] Setup basics with the Auth page
- [x] [FE] Setup readme
- [x] Create or update unit tests

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### CODE REVIEW

File: src/applications/check-in/travel-claim/README.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+What we know
+
+- Using v2 of the api for low auth
+- GET from vets-api for appointment and patient data
+- POSTing to a vets-api BTSSS endpoint to file a travel claim
+
+### How to run locally
+
+Follow the standard directions to run the app. The API needs to be running in order to run the app locally. Currently I would use the mock api in `src/applications/pre-check-in/api/local-mock-api` using the directions in the [README](https://github.com/department-of-veterans-affairs/vets-website/blob/main/README.md#running-a-mock-api-for-local-development). This makes development easier since creating a valid token is tedious.
```
Comment (brianseek) [inline src/applications/check-in/travel-claim/README.md:36]: Not a change that you need to make but I would suggest saying TBD on this until we build out the mocks

File: src/applications/check-in/travel-claim/README.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+- Create the codespace as above and wait for it to build
+- Start the mock API in a Codespace terminal: `yarn mock-api --responses path/to/mock/api.js`
+- Start the app in another Codespace terminal: `yarn watch --env.api="https://${CODESPACE_NAME}-3000.githubpreview.dev" --env.entry pre-check-in`
+- Go to the "Ports" tab and make both port 3000 and 3001 public by right-clicking and selecting Privacy -> Public:
+  <img width="999" alt="Screen Shot 2021-10-13 at 2 35 24 PM" src="https://user-images.githubusercontent.com/101649/137209007-c38ea216-1450-47f5-8d4a-7873f5cf82e1.png">
+- Hover over the "Local Address" on the line for port 3001 and click the globe icon to open the public URL in your browser.
+
+### Where is the data coming from?
+
+- Patient and Appointment data are coming from Profile service.
```
Comment (brianseek) [inline src/applications/check-in/travel-claim/README.md:63]: Appointment data is coming from veTEXT

File: src/applications/check-in/travel-claim/app-entry.jsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import 'platform/polyfills';
+import '../sass/check-in.scss';
+
+import startApp from 'platform/startup';
```
Comment (brianseek) [inline src/applications/check-in/travel-claim/app-entry.jsx:4]: we should see if using the new import methods work for this app.

**Review by brianseek — CHANGES_REQUESTED** [review]
just a couple little things

File: src/applications/check-in/travel-claim/app-entry.jsx (hunk 3 above)
Comment (wullaski) [reply src/applications/check-in/travel-claim/app-entry.jsx:4]: I think we would have to refactor our router a bit for this to work. I got stuck trying.

Comment (brianseek) [reply src/applications/check-in/travel-claim/app-entry.jsx:4]: Ah, fair enough!

**Review by brianseek — APPROVED** [review]
🚀

**Pull request merged**